src/less/generic.less

```diff
@@ -44,3 +44,7 @@ a.text-colored,
     font-size: 12px;
   }
 }
+
+.ant-tag.not-clickable {
```

Comment:
can we rename this to `.unset-cursor`?

and if it doesn't override the `.ant-tag` styling on it's own you should add it under `#root` above.